cmd/zpool/zpool_main.c

```diff
@@ -1766,7 +1766,7 @@ print_import_config(const char *name, nvlist_t *nv, int namewidth, int depth,
  */
 static void
 print_logs(status_cbdata_t *cb, zpool_handle_t *zhp, nvlist_t *nv,
-    int namewidth, boolean_t verbose, int name_flags)
+    int namewidth, int name_flags)
```

Comment:
Using a passed `cb` to infer the previous `verbose` flag works with the existing code but really isn't the best interface.  Leaving this as a function argument seems preferable.  Or better yet a new `status_cbdata_t.cb_status` flag could be added and the existing `cbp` passed in `status_callback`.  That might allow you to drop some of the existing arguments `print_logs`.